src/test/java/core/basesyntax/handlers/ReturnHandlerTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.handlers;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReturnHandlerTest {
+    private static Activity activity;
+    private static final int POSITIVE_CURRENT_COUNT = 10;
+    private static final int NEGATIVE_CURRENT_COUNT = -15;
```

Comment:
you reuse same constants in several classes, make them public and you won't need to declare them 3 times